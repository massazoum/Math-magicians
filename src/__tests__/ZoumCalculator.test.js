import React from 'react';
import renderer from 'react-test-renderer';
import Affiche from '../components/ZoumCalculator';

test('renders Affiche component correctly', () => {
  const tree = renderer
    .create(<Affiche />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
