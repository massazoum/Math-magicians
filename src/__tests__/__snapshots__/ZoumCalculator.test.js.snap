// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Affiche component correctly 1`] = `
<div
  className="layoutCaculator"
>
  <h2>
    Let s do some math 
  </h2>
  <div
    className="Calculatrice"
  >
    <div
      className="affichage"
    >
      0
    </div>
    <div>
      <button
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="ColorBtn"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
    </div>
    <div>
      <button
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="ColorBtn"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
    </div>
    <div>
      <button
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="ColorBtn"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
    </div>
    <div>
      <button
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="ColorBtn"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
    <div>
      <button
        className="zero"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="ColorBtn"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
